src/docs/environment.mdx

```diff
@@ -170,6 +172,106 @@ If you would like to be able to run `devserver` outside of your root checkout, y
   displayed correctly.
 </Alert>
 
+## Setting up Getsentry
+
+<Alert title="Corporate Users Only" level="warning">
+  Only Sentry employees have access to getsentry.
+</Alert>
+
+Now that you have sentry all set up, it's time to set up getsentry. Getsentry is
+the application we deploy to production and contains additional functionality
+only offered in our SaaS offering.
+
+Let's start off by cloning the `getsentry` repository to be adjacent with your
+sentry repository:
+
+```bash
+# Go to where you have sentry and clone getsentry.
+cd /path/to/sentry
+cd ..
+git clone git@github.com:getsentry/getsentry.git
+```
+
+It's necessary to keep getsentry in an adjacent directory (it's expected by
+`install-develop-requirements` in the getsentry Makefile). For example, if you
+did a `ls ~/code` you'd see something like:
+
+```bash
+sentry/   getsentry/
+```
+
+Next, create a virtualenv for `getsentry`, and install its dependencies:
```

Comment:
I would replace all this with simply: Run `make develop` and follow the instructions from there.

Reason being it's redundant now that `make develop` will hold your hand:

```
You don't seem to have a virtualenv. Please create one by running: python -m virtualenv .venv
make: *** [ensure-venv] Error 1
```